open-webui.yaml

```diff
@@ -0,0 +1,105 @@
+package:
+  name: open-webui
+  version: 0.6.13
+  epoch: 0
+  description: User-friendly AI Interface (Supports Ollama, OpenAI API, ...)
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - ffmpeg
+      - glib
+      - libglvnd
+      - libice
+      - libsm
+      - libstdc++
+      - libx11
+      - libxcb
+      - libxext
+      - python-${{vars.py-version}}-base
+  options:
+    # Don't depend on external libraries included in virtual environments
+    no-depends: true
+    # Don't resolve libraries installed in virtual environments as providers
+    no-provides: true
+
+vars:
+  py-version: 3.11
+
+environment:
+  contents:
+    packages:
+      - busybox
+      # Does not support NodeJS 23
+      - nodejs-22
+      - npm
+      - python-${{vars.py-version}}
+      - uv
+  environment:
+    NODE_OPTIONS: --max_old_space_size=8192
+    SOURCE_DATE_EPOCH: 315532900
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 53764fe64884da147359e54ed6d9607fe57f1600
+      repository: https://github.com/open-webui/open-webui
+      tag: v${{package.version}}
+
+  - uses: patch
+    with:
+      patches: open-webui-dependency-update.patch
+
+  - name: Create venv
+    runs: |
+      # Create virtual env
+      uv venv venv --seed
+      source venv/bin/activate
+
+      # Install open-webui
+      uv pip install --upgrade pip
+      uv pip install .
+      uv pip uninstall pip
+
+      # Cleanup installation
+      find venv -type d -name '__pycache__' -exec rm -rf {} +
+      find venv -type f -name '*.pyc' -exec rm -rf {} +
+      find venv -type d -name "tests" -exec rm -rf {} +
+
+      # Use Python in virtual environment
+      sed -i "s|/home/build/venv|/usr/share/${{package.name}}|g" venv/pyvenv.cfg
+      sed -i "s|/home/build/venv|/usr/share/${{package.name}}|g" venv/bin/*
+
+      # Install virtual env
+      mkdir -p ${{targets.contextdir}}/usr/share/${{package.name}}/
+      cp -r venv/* ${{targets.contextdir}}/usr/share/${{package.name}}/
+
+      # Symlink binaries
+      mkdir -p ${{targets.contextdir}}/usr/bin
+      for bin in ${{targets.contextdir}}/usr/share/${{package.name}}/bin/*
+      do
+        # Don't link venv python - clashes with runtime python dep
+        if [[ $bin != *"python"* ]]
+        then
+          ln -sf "/usr/share/${{package.name}}/bin/${bin##*/}" "${{targets.contextdir}}/usr/bin/${bin##*/}"
+        fi
+      done
+
+test:
+  pipeline:
+    - uses: test/daemon-check-output
+      with:
+        expected_output: |
+          Creating knowledge table
+          v${{package.version}} - building the best AI user interface.
+          Started server process
+          Waiting for application startup.
+        start: open-webui serve
+        timeout: 300
```

Comment:
Let's have a `post:` field here, ensure port is up with `wait-for-it` and with a simple `curl` test to `grep` web content?